src/Renci.SshNet/Security/Cryptography/CipherDigitalSignature.cs

```diff
@@ -1,90 +0,0 @@
-﻿using System;
-using Renci.SshNet.Common;
-
-namespace Renci.SshNet.Security.Cryptography
-{
-    /// <summary>
-    /// Implements digital signature where where asymmetric cipher is used.
-    /// </summary>
-    public abstract class CipherDigitalSignature : DigitalSignature
-    {
-        private readonly AsymmetricCipher _cipher;
-        private readonly ObjectIdentifier _oid;
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="CipherDigitalSignature"/> class.
-        /// </summary>
-        /// <param name="oid">The object identifier.</param>
-        /// <param name="cipher">The cipher.</param>
-        protected CipherDigitalSignature(ObjectIdentifier oid, AsymmetricCipher cipher)
-        {
-            if (cipher is null)
-            {
-                throw new ArgumentNullException(nameof(cipher));
-            }
-
-            _cipher = cipher;
-            _oid = oid;
-        }
-
-        /// <summary>
-        /// Verifies the signature.
-        /// </summary>
-        /// <param name="input">The input.</param>
-        /// <param name="signature">The signature.</param>
-        /// <returns>
-        /// <see langword="true"/> if signature was successfully verified; otherwise <see langword="false"/>.
-        /// </returns>
-        public override bool Verify(byte[] input, byte[] signature)
-        {
-            var encryptedSignature = _cipher.Decrypt(signature);
-            var hashData = Hash(input);
-            var expected = DerEncode(hashData);
-            return expected.IsEqualTo(encryptedSignature);
```

Comment:
There could be side-channel timing attack here. Removing this file and use BCL implementation should fix the issue.